site/server/app/com/fortysevendeg/exercises/persistence/domain/UserProgressQueries.scala

```diff
@@ -24,7 +24,7 @@ object SaveUserProgress {
       method:       String,
       version:      Int,
       exerciseType: ExerciseType,
-      args:         Option[String],
+      args:         Option[List[String]],
```

Comment:
Can't a postgress nullable value map to an empty List instead of double nesting type constructors with empty capable semantics?